packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol

```diff
@@ -96,7 +102,7 @@ contract L2ToL1MessagePasser is Semver {
     ) public payable {
         bytes32 withdrawalHash = Hashing.hashWithdrawal(
```

Comment:
I think you should also add logic to `Hashing.hashWithdrawal` that observes the version

Reply:
Feels unnecessary for now since we don't actually have a second version. It would be easy to add in the future if/when we have a second version.